cmd/misc/index-old-eth1-blocks/main.go

```diff
@@ -1,207 +0,0 @@
-package main
-
-import (
-	"context"
-	"eth2-exporter/db"
-	"eth2-exporter/rpc"
-	"eth2-exporter/types"
-	"eth2-exporter/utils"
-	"eth2-exporter/version"
-	"flag"
-	"fmt"
-	"net/http"
-	"strconv"
-	"strings"
-
-	"github.com/coocood/freecache"
-	_ "github.com/jackc/pgx/v4/stdlib"
-	"github.com/sirupsen/logrus"
-
-	_ "net/http/pprof"
-)
-
-func main() {
-	erigonEndpoint := flag.String("erigon", "", "Erigon archive node enpoint")
-	block := flag.Int64("block", 0, "Index a specific block")
-
-	startBlock := flag.Int64("blocks.start", 0, "Block to start indexing")
-	endBlock := flag.Int64("blocks.end", 0, "Block to finish indexing")
-
-	concurrencyData := flag.Int64("data.concurrency", 30, "Concurrency to use when indexing data from bigtable")
-	batchSize := flag.Int64("data.batchSize", 1000, "Batch size")
-
-	bigtableProject := flag.String("bigtable.project", "", "Bigtable project")
-	bigtableInstance := flag.String("bigtable.instance", "", "Bigtable instance")
-
-	transformerFlag := flag.String("transformers", "", "Comma separated list of transformer functions")
-
-	versionFlag := flag.Bool("version", false, "Print version and exit")
-
-	configPath := flag.String("config", "", "Path to the config file, if empty string defaults will be used")
-
-	flag.Parse()
-
-	if *versionFlag {
-		fmt.Println(version.Version)
-		return
-	}
-
-	logrus.Infof("transformerFlag: %v %s", *transformerFlag, *transformerFlag)
-	transformerList := strings.Split(*transformerFlag, ",")
-	if len(transformerList) == 0 {
-		utils.LogError(nil, "no transformer functions provided", 0)
-		return
-	}
-
-	cfg := &types.Config{}
-	err := utils.ReadConfig(cfg, *configPath)
-	if err != nil {
-		utils.LogError(err, "error reading config file", 0)
-	}
-	utils.Config = cfg
-	logrus.WithField("config", *configPath).WithField("version", version.Version).WithField("chainName", utils.Config.Chain.Config.ConfigName).Printf("starting")
-
-	// enable pprof endpoint if requested
-	if utils.Config.Pprof.Enabled {
-		go func() {
-			logrus.Infof("starting pprof http server on port %s", utils.Config.Pprof.Port)
-			logrus.Info(http.ListenAndServe(fmt.Sprintf("localhost:%s", utils.Config.Pprof.Port), nil))
-		}()
-	}
```

Comment:
This feature now is not available anymore. Is that intended?

Reply:
yes - not needed for usecase